Cargo.toml

```diff
@@ -58,7 +56,9 @@ quircs = "0.10.0"
 symmetric-cipher = { git = "https://github.com/input-output-hk/chain-wallet-libs.git", branch = "master" }
 graphql_client = "0.10"
 gag = "1"
-vit-servicing-station-lib = { git = "https://github.com/danielsanchezq/vit-servicing-station.git", branch = "master" }
+vit-servicing-station-lib = { git = "https://github.com/input-output-hk/vit-servicing-station.git", branch = "master" }
+quickcheck = "0.9"
```

Comment:
What do you think to switch it to the `protest` as it is first appearance of the `quickcheck` ?

Reply:
could be a good idea